tools/scaffold/template/pluraldatasource.tmpl

```diff
@@ -0,0 +1,66 @@
+package {{.NameLowerNoSpaces}}
+
+import (
+	"context"
+
+	"github.com/hashicorp/terraform-plugin-framework/datasource"
+	"github.com/hashicorp/terraform-plugin-framework/datasource/schema"
+	"github.com/hashicorp/terraform-plugin-framework/types"
+	"github.com/mongodb/terraform-provider-mongodbatlas/internal/config"
+	"github.com/mongodb/terraform-provider-mongodbatlas/internal/common/dsschema"
+)
+
+var _ datasource.DataSource = &{{.NameCamelCase}}sDS{}
```

Comment:
in the general case s will work for plural resources, for example it doesn't work for data_source_search_indexes.
don't know if allow an optional param that defaults to the name with s,  or maybe we don't need this at the moment and we can change it later.

Reply:
yeah I believe given it is an edge case, would not include additional logic at the moment